doc/apireference.rst

```diff
@@ -54,6 +54,7 @@ All 2-d operations require a **Workplane** object to be created.
 	Workplane.threePointArc
 	Workplane.sagittaArc
 	Workplane.radiusArc
+  Workplane.tangentArcTo
```

Comment:
Should be `Workplane.tangentArcEndpoint`

Reply:
@marcus7070 I would go ahead and make the change.